src/pages/accounts/AccountDetails.tsx

```diff
@@ -1,451 +1,279 @@
-    contacts: string;
-    status: string;
-    source: string;
-    address_line: string;
-    street: string;
-    city: string;
-    state: string;
-    postcode: string;
-    country: string;
-    tags: [];
-    company: string;
-    probability: string;
-    industry: string;
-    skype_ID: string;
-    file: string;
+const AccountDetails: React.FC = () => {
+  const { accountId } = useParams<{ accountId: string }>();
+  const [account, setAccount] = useState<Account | null>(null);
+  const { userRole, userId } = useMyContext();
+  const [data, setData] = useState<any[]>([]);
 
-    close_date: string;
-    organization: string;
-    created_from_site: boolean;
-    id: string;
-    teams: [];
-    leads: string;
+  useEffect(() => {
+    if (accountId) {
+      fetchAccount();
+      getAccounts();
+    }
+  }, [accountId]);
 
-};
-export const AccountDetails = (props: any) => {
-    const { state } = useLocation()
-    console.log(state)
-    const navigate = useNavigate()
-    const { userRole , profileId, userId} = useMyContext();
+  const getAccounts = async () => {
+    const Header = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: localStorage.getItem('Token'),  // Fallback to undefined if null
+      org: localStorage.getItem('org'),               // Fallback to undefined if null
+    };
+  
+    try {
+      const res = await fetchData(`${AccountsUrl}`, 'GET', null as any, Header); // Properly passing null for body
+      if (!res.error) {
+        setData(res || []); // Setting the fetched data or empty array
+      }
+    } catch (error) {
+      console.error('Error fetching data:', error); // Logging any errors that occur
+    }
+  };
+  
 
-    const [accountDetails, setAccountDetails] = useState<response | null>(null)
-    const [usersDetails, setUsersDetails] = useState<Array<{
-        user_details: {
-            email: string;
-            id: string;
-            profile_pic: string;
-        }
-    }>>([]);
-    const [selectedCountry, setSelectedCountry] = useState([])
-    const [attachments, setAttachments] = useState([])
-    const [tags, setTags] = useState([])
-    const [countries, setCountries] = useState<string[][]>([])
-    const [source, setSource] = useState([])
-    const [status, setStatus] = useState([])
-    const [industries, setIndustries] = useState([])
-    const [contacts, setContacts] = useState([])
-    const [users, setUsers] = useState([])
-    const [teams, setTeams] = useState([])
-    const [leads, setLeads] = useState([])
-    const [comments, setComments] = useState([])
-    const [commentList, setCommentList] = useState('Recent Last')
-    const [note, setNote] = useState('')
+  const fetchAccount = async () => {
+    const Header = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: localStorage.getItem('Token'),
+      org: '803a0394-5f34-4e3f-9d48-dbe7df156bd0',
```

Comment:
‼️ **Important** ‼️ 
This line should be replaced by 
`org: localStorage.getItem('org'),`
because everyone has a different token for org, so it should be taken dynamically from localStorage.